### PR TITLE
Test/sadpathtests

### PR DESCRIPTION
**Pull request by Billwilke42**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

-  No sad path tests or error handling for lives, answer, header or question component. 

## What is the new behavior?

- Sad path tests and error handling for above listed components.

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/GameOver/GameOver.test.js [hunk 1]
```diff
@@ -49,9 +47,25 @@ describe('Game Over', () => {
...
+			</BrowserRouter>);
 
-})
\ No newline at end of file
+			const errorMsg = getByText('Error: You have no data, how did you get here?');
+			
+      expect(errorMsg).toBeInTheDocument();
+    })  
+})
+
```
Comment (KyleWong2510) [inline src/GameOver/GameOver.test.js:71]: Nice job with this error handling messages!

File: src/Lives/Lives.test.js [hunk 2]
```diff
@@ -30,4 +30,17 @@ describe('Lives', () => {
...
+					<Provider store={store}>
+            <Lives />
+					</Provider>
+        </BrowserRouter>);
+        
+  
+    const errorMessage = getByText('Error: No Lives Were Given')
+
+    expect(errorMessage).toBeInTheDocument();
+	});
```
Comment (KyleWong2510) [inline src/Lives/Lives.test.js:45]: more indent issues?

Comment (KyleWong2510) [reply src/Lives/Lives.test.js:45]: Nice job on those sad paths!

File: src/Answers/Answers.js [hunk 3]
```diff
@@ -27,7 +28,9 @@ const Answers = (props) => {
     <section className='answer-container'>
       {mappedAnswers}
     </section>
-  )
+  )} else {
+    return <p>Error: No Answers Provided</p>
```
Comment (lilianaweimer) [inline src/Answers/Answers.js:32]: Nice error handling!

File: src/GameOver/GameOver.js [hunk 4]
```diff
@@ -6,14 +6,20 @@ import { reset } from '../actions';
...
+      <section className='game-over nes-balloon from-left'>
+        <h2>GAME OVER</h2>
+        <p>Score: {props.score}</p>
+        {props.lives > 0 && <p>Lives Left: {props.lives}</p>}
+        <Link to='/'><button className='nes-btn' type='submit' onClick={props.reset}>Home</button></Link>
+      </section>
+    )} 
+    if(!props.playerName) {
+      return (
+        <p>Error: You have no data, how did you get here?</p>
```
Comment (lilianaweimer) [inline src/GameOver/GameOver.js:20]: Love this error message!

File: src/Lives/Lives.test.js (hunk 2 above)
Comment (lilianaweimer) [reply src/Lives/Lives.test.js:45]: Ah no!! The indents again :(

**Pull request merged**